packages/ui-library/src/components/radio/index.test.ts

```diff
@@ -54,6 +55,25 @@ describe('blr-radio', () => {
     expect(errorClassName).to.contain('error');
   });
 
+  it('error Icon is set to undefined', async () => {
+    const element = await fixture(
+      BlrRadioRenderFunction({
+        ...sampleParams,
+        hasError: true,
+        errorIcon: undefined,
+      })
+    );
+
+    const labelWrapper = querySelectorDeep('.label-wrapper', element.getRootNode() as HTMLElement);
+    const captionWrapper = querySelectorDeep('.caption-wraper', labelWrapper?.getRootNode() as HTMLElement);
+    const formCaptions = querySelectorAllDeep('.blr-form-caption', captionWrapper?.getRootNode() as HTMLElement);
+    const formCaptionError = querySelectorDeep('.blr-form-caption', formCaptions[1] as HTMLElement);
```

Comment:
Isn't it possible to target the error inside the form-caption directly just like in other cases? The querySelectorAllDeep is useful if both the hint and error needs to be tested. 
Note: hasHint can be set to false (if not already set in sampleParams) to assure that only one icon exist inside the form-caption.